Source/gnustep-gui.pc.in

```diff
@@ -0,0 +1,6 @@
+Name: gnustep-gui
```

Comment:
Why is this file in Source and not in the main directory? Or a separate subdirectory?